configs/projects/puppet-agent.rb

```diff
@@ -1,34 +1,68 @@
 project "puppet-agent" do |proj|
-  # Project level settings our components will care about
-  proj.setting(:install_root, "/opt/puppetlabs")
-  proj.setting(:prefix, File.join(proj.install_root, "puppet"))
+  platform = proj.get_platform
 
-  if platform.is_eos?
-    proj.setting(:sysconfdir, "/persist/sys/etc/puppetlabs")
-    proj.setting(:link_sysconfdir, "/etc/puppetlabs")
-  elsif platform.is_osx?
-    proj.setting(:sysconfdir, "/private/etc/puppetlabs")
+  # Project level settings our components will care about
+  # Windows has its own seperate layout
+  if platform.is_windows?
+    # Our install prefix can't have spaces in it, so we take advantage of short cuts.
+    # However, in order to take advantage of these shortcuts, we need to explicitely
+    # create the directories.
+    proj.setting(:install_root, "#{platform.drive_root}/Progra~1/Puppet~1")
+    proj.directory "#{platform.drive_root}/Program Files/Puppet Labs"
```

Comment:
@joshcooper I'm pretty sure this is working now! The only issue is that the install prefix is changing from `Puppet` to `puppet`, and I'm not sure why. We're setting it to upper case puppet on line 13.